src/components/ebay-button/README.md

```diff
@@ -1,12 +1,15 @@
 # ebay-button
```

Comment:
I'm not a huge of the repetition below, but if we keep it, we could probably remove top level headings like this, and move all the other headings up a notch.